test/parallel/test-net-listening.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Done, all those changes will be on my next commit, as soon as I match it with the contribution guidelines.